EventSite/Domain/Model/Sponsor.cs

```diff
@@ -16,6 +16,11 @@ public class Sponsor {
         public SponsorStatus Status { get; set; }
         public DateTimeOffset? DonatedOn { get; set; }
 
+        public Sponsor()
+        {
+            this.Link = "http://";
```

Comment:
Keith was complaining about having to put the "http://" in the field for sponsors. My plan was to add server side validation, just ran out of time today. You can take it out it and I'll put something legit in tomorrow.


Sent from Outlook






On Tue, Feb 9, 2016 at 6:16 PM -0800, "Jesse Taber" <notifications@github.com> wrote:












In EventSite/Domain/Model/Sponsor.cs:
> @@ -16,6 +16,11 @@ public class Sponsor {
>          public SponsorStatus Status { get; set; }
>          public DateTimeOffset? DonatedOn { get; set; }
>  
> +        public Sponsor()
> +        {
> +            this.Link = "http://";



@jamescrowley321 what's up with this?



—
Reply to this email directly or view it on GitHub.